templates/shared-memory-service.json

```diff
@@ -305,6 +318,59 @@
                         }
                      }
                   ],
+                  "initContainers": [
+                     {
+                        "name": "pem-to-keystore",
+                        "image": "registry.access.redhat.com/redhat-sso-7/sso71-openshift:1.1-16",
```

Comment:
That's an awesome trick! It deserves a blog post!

Reply:
I actually discovered this via a blog post: https://developers.redhat.com/blog/2017/11/22/dynamically-creating-java-keystores-openshift/